src/viewmodel/prototype/get.js

```diff
@@ -62,6 +63,12 @@ export default function Viewmodel$get ( keypath, options ) {
 		value = wrapped.get();
 	}
 
+	if ( keypath.isRoot && options.fullRootGet ) {
+		for ( key in this.mappings ) {
+			value[ key ] = this.mappings[ key ].getValue();
+		}
```

Comment:
Having a separate method would certainly make *our* lives easier! Not sure about users though.

The question for me is whether we should expect this to work or not:

```js
var data = ractive.get();
data.foo = 'bar'; // could argue that this is unsupported, shd be ractive.set('foo','bar')
```

But maybe that's a widespread pattern and we shouldn't break it. Not sure.